### PR TITLE
Update CI workflow

### PR DESCRIPTION
**Pull request by szepeviktor**
@lptn Please see comments below.

### CODE REVIEW

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -6,19 +6,18 @@ on:
 
 jobs:
   test:
-    runs-on: ${{ matrix.os }}
+    runs-on: ubuntu-latest
     strategy:
       fail-fast: true
       matrix:
-        os: [ubuntu-latest]
```
Comment (szepeviktor) [inline .github/workflows/run-tests.yml:13]: Remove variable with one value only.

File: .github/workflows/run-tests.yml [hunk 2]
```diff
@@ -6,19 +6,18 @@ on:
...
-        php: [8.1, 8.2]
+        php: ['8.1', '8.2']
```
Comment (szepeviktor) [inline .github/workflows/run-tests.yml:13]: Prevent PHP versions from being YAML floating point values.

File: .github/workflows/run-tests.yml [hunk 3]
```diff
@@ -39,10 +38,7 @@ jobs:
       - name: Install dependencies
         run: |
           composer config "http-basic.nova.laravel.com" "${{ secrets.NOVA_USERNAME }}" "${{ secrets.NOVA_4_LICENSE_KEY }}" --ansi
-          composer require "laravel/framework:${{ matrix.laravel }}" "orchestra/testbench:${{ matrix.testbench }}" --no-interaction --no-update
-          composer update ${{ matrix.dependency-version }} --prefer-dist --no-interaction --no-suggest --ansi
-        env:
-          COMPOSER_AUTH: ${{ secrets.COMPOSER_AUTH }}
```
Comment (szepeviktor) [inline .github/workflows/run-tests.yml:45]: `COMPOSER_AUTH` is unused.

File: .github/workflows/run-tests.yml [hunk 4]
```diff
@@ -39,10 +38,7 @@ jobs:
...
+          composer update ${{ matrix.dependency-version }} --prefer-dist --no-interaction --ansi --with="laravel/framework:${{ matrix.laravel }}" --with="orchestra/testbench:${{ matrix.testbench }}"
```
Comment (szepeviktor) [inline .github/workflows/run-tests.yml:41]: Composer v2 introduced `--with`

File: .github/workflows/run-tests.yml [hunk 5]
```diff
@@ -39,10 +38,7 @@ jobs:
...
 
       - name: Execute tests
-        run: composer test
+        run: composer run test
```
Comment (szepeviktor) [inline .github/workflows/run-tests.yml:44]: Distinguish Composer script names from Composer commands.

**Review by lptn — APPROVED** [review]
great improvements, as usually @szepeviktor ! 
✨

**Pull request merged**